doc/guides/tips-and-troubleshooting-for-working-with-the-code.md

```diff
@@ -0,0 +1,26 @@
+# Tips & Troubleshooting for working with the `node` code 
+
+## Tips
+
+1. If you build the code often, you should check out [`ninja`]. You may
```

Comment:
I tend to ` wrap program names, but in this case, I agree it makes more sense not to